### ISSUE TITLE
Sickchill package stopeed after last sickchill GUI update

### ISSUE THREAD

**Issue by kretzu77** [SynoCommunity/spksrc#5393]
### Is this a new Bug?

- [X] I checkd that the bug hasn't been reported before

### Package Name

sickchill

### Package Version

20211110-4

### Device Model

Synology DS1621xs+

### Device Architecture

x86_64

### Firmware Version

DSM 7.1

### What happened?

![Untitled](https://user-images.githubusercontent.com/30404637/185353596-7176ea12-cd3c-492a-b6d3-ecf174ce7ce5.jpg)

After las GUI update, sickchill package don't start anymore
I tried manual Run, but stopped immediately
[sickchill.log](https://github.com/SynoCommunity/spksrc/files/9372091/sickchill.log)
 

### Reproduction steps

```bash
1. Sickchill GUI update
2. Manual Run sickchill
3. ssh with command sudo chown -R sc-sickchill:sickchill /volume1/@appstore/sickchill/env with no luck
4. Even reboot NAS
```

### Install Log

```log
2022/08/18 11:32:33	start sickchill 20211110-4 Begin start-stop-status stop
2022/08/18 11:32:33	start sickchill 20211110-4 End start-stop-status stop ret=[0]
2022/08/18 11:32:34	start sickchill 20211110-4 Begin start-stop-status start
2022/08/18 11:32:35	start sickchill 20211110-4 End start-stop-status start ret=[0]
2022/08/18 11:32:46	start sickchill 20211110-4 Begin start-stop-status stop
2022/08/18 11:32:46	start sickchill 20211110-4 End start-stop-status stop ret=[0]
2022/08/18 11:32:47	start sickchill 20211110-4 Begin start-stop-status start
2022/08/18 11:32:48	start sickchill 20211110-4 End start-stop-status start ret=[0]
2022/08/18 11:36:02	start sickchill 20211110-4 Begin start-stop-status stop
2022/08/18 11:36:02	start sickchill 20211110-4 End start-stop-status stop ret=[0]
2022/08/18 11:36:03	start sickchill 20211110-4 Begin start-stop-status start
2022/08/18 11:36:04	start sickchill 20211110-4 End start-stop-status start ret=[0]
2022/08/18 11:39:28	start sickchill 20211110-4 Begin start-stop-status stop
2022/08/18 11:39:28	start sickchill 20211110-4 End start-stop-status stop ret=[0]
2022/08/18 11:39:29	start sickchill 20211110-4 Begin start-stop-status start
2022/08/18 11:39:30	start sickchill 20211110-4 End start-stop-status start ret=[0]
```

**Comment by BenjV** [comment]
You cannot run a package from the commandline that way.
Packages need a number of environment variables to be set by the package center.
The way to start a package from the commandline is via:
Synopkg start "packagename + parameters"

But it should start normal, so the update broke something you should ask the sickchill developers, not the SynoCommunity

**Comment by hgy59** [comment]
fixed with #5458
